docs/www/configuration.md

```diff
@@ -751,7 +738,7 @@ Here is a more comprehensive view of the configration so that you can see all of
 | `max_version` | max redpanda compat version | 1 |
```

Comment:
Should these also be removed to align with the entries removed above?

Might be good to fix the indentation here, too.

Reply:
Good spot, removed.